src/main/java/org/datacleaner/kettle/configuration/DataCleanerConfigurationDialog.java

```diff
@@ -184,7 +183,6 @@ public void widgetSelected(SelectionEvent event) {
             }
         } catch (Throwable e) {
             // Do nothing If the file doesn't exit 
-
```

Comment:
Nothing worsened by this PR, but I don't like swallowed exceptions either. At the very least we should log it, but if it is in deed "just" that a file does not exist, then it can be handled with a simple check instead of catching a very broad class of exceptions (throwables even).